docs/magento2/detect-admin-users.mdx

```diff
@@ -78,14 +78,10 @@ export default {
 ```
 
 ```js title="server/modules/admin-data/loaders.js"
-import { makeUserClientFromRequest } from "server/modules/magento1/core/factories";
-import { AdminDataLoader } from "./loaders";
-
 export const AdminDataLoader = (axiosInstance) => {
   return {
-    loadData: async () => {
-      const response = await axiosInstance.get("/admin-custom-endpoint");
-      return response.data;
+    loadData: () => {
+      return axiosInstance.get("/admin-custom-endpoint");
```

Comment:
**question**: the returned data has changed here (`response.data` vs `response`), is it wanted?

Reply:
true thanks for seeing that :)